### DISCUSSION TITLE
Is there an up-to-date documentation and downloads?

### DISCUSSION THREAD

**Discussion by ohadvano** [esnet/iperf#1481] [Q&A]
Is there an up-to-date documentation and release downloads?
Seems like https://iperf.fr/iperf-doc.php is obsolete and the release downloads are not being updated.
Is there an alternative? Also is there any known source for downloading executable artifacts for Windows?
Thanks for helping

**Comment by tinyboxvk** [comment]
See https://github.com/esnet/iperf/issues/1475#issuecomment-1447179617

In the meantime, I have a GitHub Action set up to build the latest iperf3 binary. Feel free to use it if you don't want to set up Cygwin yourself.

https://github.com/tinyboxvk/iperf-windows-builds/releases/

**Reply by megapro17** [reply]
There were native builds that took much smaller space, around 100 kbytes https://github.com/tniessen/iperf-windows

**Comment by ohadvano** [comment]
Thanks! Do you know if there's plan to update the docs?

**Reply by swlars** [reply]
Do you mean the documents on iperf.fr? We don't have any access to it, and we haven't been able to contact the maintainers. 

There is documentation here: https://software.es.net/iperf/.

**Comment by cschar** [comment]
The #1 and #2 google serach results are iperf.fr... only the 3rd result is this github repo!


Is this site 
https://iperf.fr/iperf-download.php

hosting the same software known as 'iperf' that this github repository advertises
<img width="454" alt="Screenshot 2024-07-30 at 4 48 17 PM" src="https://github.com/user-attachments/assets/8fc92118-b53d-4370-8359-000657db162f">

It is directing users to a strange domain "files.budman.pw"....

What do we think about clarifying the role this site is playing? It doesnt have the best appearance of legitimacy IMO.
